scripts/main.py

```diff
@@ -292,14 +292,19 @@ def parse_arguments():
 
 # Interaction Loop
 while True:
+    # Setting up the token limit and model type according to model type
+    token_limit = (cfg.fast_token_limit if cfg.gpt3only else cfg.smart_token_limit)
+    model = (cfg.fast_llm_model if cfg.gpt3only else cfg.smart_llm_model)
```

Comment:
This changes the Default to GPT4 only, which is not the intended design of the program.

We want GPT3 as default for the main interaction loop.

Reply:
That said, a `gpt4only` option would be a nice feature that has been requested, though that would be a different PR.

Reply:
Closed this PR in favour of #733